packages/@aws-cdk/aws-ecs/lib/drain-hook/instance-drain-hook.ts

```diff
@@ -57,6 +61,11 @@ export class InstanceDrainHook extends cdk.Construct {
     super(scope, id);
 
     const drainTime = props.drainTime || cdk.Duration.minutes(5);
+    const clusterArn = cdk.Stack.of(this).formatArn({
+      service: 'ecs',
+      resource: 'cluster',
+      resourceName: props.clusterName,
+    });
```

Comment:
We still need `clusterArn` in addition to `clusterName` but we don't need the `cluster` construct. We can simply formate the ARN from the `clusterName`.